test/functional/rpc_net.py

```diff
@@ -26,6 +26,11 @@
 )
 from test_framework.wallet import MiniWallet
 
+# Address manager size constants as defined in addrman_impl.h
+ADDRMAN_NEW_BUCKET_COUNT = 1 << 10
+ADDRMAN_TRIED_BUCKET_COUNT = 1 << 8
+ADDRMAN_BUCKET_SIZE = 1 << 6
+
```

Comment:
If I re-touch this, I'll move the constants to `test/functional/test_framework/netutil.py` as in https://github.com/bitcoin/bitcoin/pull/28176#issuecomment-1741842516

Reply:
#28176 was merged, you might want to rebase on it to avoid duplication.